server/src/concepts/offer/model/OfferValidation.js

```diff
@@ -0,0 +1,32 @@
+import Joi from "joi-oid";
+
+const validateCreateOffer = (offer) => {
+  const schema = Joi.object({
+    title: Joi.string().min(3).max(50).required(),
+
+    description: Joi.string().max(500).required(),
+
+    city: Joi.string().max(58).required(),
+
+    phoneNumber: Joi.string().min(9).max(9).required(),
+
+    photos: Joi.array().items(
+      Joi.object({
+        url: Joi.string().required(),
+        isMainPhoto: Joi.boolean().required(),
+      })
+    ),
+
+    category: Joi.objectId().required(),
+
+    height: Joi.objectId().required(),
+
+    price: Joi.number().required(),
```

Comment:
We can think about better validation about price, maybe that it cannot be less than 0 and specify the number of decimal places ref to docs:
https://joi.dev/api/?v=17.6.0#numberminlimit
https://joi.dev/api/?v=17.6.0#numberprecisionlimit

Beside that we can "connect" this field with forExhcange because in our case when forExchange is true, price have to be 0. It is possible to validate that, ref to docs: https://joi.dev/api/?v=17.6.0#anywhencondition-options. But for now that is okey of course